net/stun/stun.go

```diff
@@ -223,7 +223,7 @@ func ParseResponse(b []byte) (tID TxID, addr netip.AddrPort, err error) {
 				return err
 			}
 			if len(a) == 16 {
-				addr6 = netip.AddrPortFrom(netip.AddrFrom16(*(*[16]byte)([]byte(a))), p)
+				addr = netip.AddrPortFrom(netip.AddrFrom16(*(*[16]byte)([]byte(a))).Unmap(), p)
```

Comment:
If we're getting rid of `addr6` then we can just do this:
```go
a, _ := netip.AddrFromSlice(slc)
addr = netip.AddrPortFrom(a.Unmap(), p)
```

Reply:
Sure, done. 

I'm trying to avoid too much cleanup with fixes at the same time, though.

There's more to be done here: the allocations from xorMappedAddress and mappedAddress are unnecessary. I can send that as a separate change, though.

Reply:
Sounds good!